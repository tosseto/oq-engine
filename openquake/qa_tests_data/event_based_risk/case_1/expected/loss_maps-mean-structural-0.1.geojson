{
    "features": [
        {
            "geometry": {
                "coordinates": [
                    81.29850006103516,
                    29.109800338745117
                ],
                "type": "Point"
            },
            "properties": {
                "asset_ref": "a0",
                "loss": 773.0548706054688
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    83.0822982788086,
                    27.90060043334961
                ],
                "type": "Point"
            },
            "properties": {
                "asset_ref": "a1",
                "loss": 316.72259521484375
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    85.7477035522461,
                    27.901500701904297
                ],
                "type": "Point"
            },
            "properties": {
                "asset_ref": "a2",
                "loss": 558.3612670898438
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    85.7477035522461,
                    27.901500701904297
                ],
                "type": "Point"
            },
            "properties": {
                "asset_ref": "a3",
                "loss": 0.0
            },
            "type": "Feature"
        }
    ],
    "oqmetadata": {
        "investigationTime": "50.0",
        "lossType": "structural",
        "poE": "0.1",
        "statistics": "mean",
        "unit": "EUR"
    },
    "oqnrmlversion": "0.4",
    "oqtype": "LossMap",
    "type": "FeatureCollection"
}