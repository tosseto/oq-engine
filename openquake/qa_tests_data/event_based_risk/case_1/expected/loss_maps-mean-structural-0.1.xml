<?xml version="1.0" encoding="utf-8"?>
<nrml
xmlns="http://openquake.org/xmlns/nrml/0.5"
xmlns:gml="http://www.opengis.net/gml"
>
    <lossMap
    investigationTime="50.0"
    lossType="structural"
    poE="0.1"
    riskInvestigationTime="50.0"
    statistics="mean"
    unit="EUR"
    >
        <node>
            <gml:Point>
                <gml:pos>
                    81.2985 29.1098
                </gml:pos>
            </gml:Point>
            <loss assetRef="a0" value="7.73055E+02"/>
        </node>
        <node>
            <gml:Point>
                <gml:pos>
                    83.0823 27.9006
                </gml:pos>
            </gml:Point>
            <loss assetRef="a1" value="3.16723E+02"/>
        </node>
        <node>
            <gml:Point>
                <gml:pos>
                    85.7477 27.9015
                </gml:pos>
            </gml:Point>
            <loss assetRef="a2" value="5.58361E+02"/>
            <loss assetRef="a3" value="0.00000E+00"/>
        </node>
    </lossMap>
</nrml>
